images/openstack/Dockerfile.ci

```diff
@@ -1,14 +1,14 @@
 # This Dockerfile is used by CI to test using OpenShift Installer against an OpenStack cloud.
 # It builds an image containing the openshift-install command as well as the openstack cli.
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 AS builder
```

Comment:
registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 is available now, shouldn't we use it?